decidim-assemblies/app/commands/decidim/assemblies/admin/update_assembly.rb

```diff
@@ -74,7 +74,28 @@ def attributes
             participatory_scope: form.participatory_scope,
             participatory_structure: form.participatory_structure,
             meta_scope: form.meta_scope,
-            show_statistics: form.show_statistics
+            show_statistics: form.show_statistics,
+            purpose_of_action: form.purpose_of_action,
+            type_of_assembly: form.type_of_assembly,
+            type_of_assembly_other: form.type_of_assembly_other,
+            date_created: form.date_created,
+            created_by: form.created_by,
+            created_by_other: form.created_by_other,
+            duration: form.duration,
+            date_of_inclusion: form.date_of_inclusion,
+            has_closed: form.has_closed,
+            closing_date: form.closing_date,
+            closing_date_reason: form.closing_date_reason,
+            internal_organisation: form.internal_organisation,
+            open_field: form.open_field,
+            public_field: form.public_field,
+            transparent_field: form.transparent_field,
+            special_features: form.special_features,
+            twitter_handler: form.twitter_handler,
+            facebook_handler: form.facebook_handler,
+            instagram_handler: form.instagram_handler,
+            youtube_handler: form.youtube_handler,
+            github_handler: form.github_handler
```

Comment:
Maybe we can group all these social handlers into an a json column?

Reply:
Organization is done as well. How do you feel better?

Reply:
Lets keep it like this then.

Reply:
Can you review it?